lte/gateway/c/core/oai/tasks/nas/esm/DefaultEpsBearerContextActivation.c

```diff
@@ -495,8 +496,8 @@ status_code_e default_eps_bearer_activate_t3485_handler(
   ue_mm_context_t* ue_mm_context = mme_app_get_ue_context_for_timer(
       ue_id, "EPS BEARER DEACTIVATE T3495 Timer");
   if (ue_mm_context == NULL) {
-    OAILOG_ERROR(
-        LOG_MME_APP,
+    OAILOG_ERROR_UE(
+        LOG_MME_APP, ue_mm_context->emm_context._imsi64,
```

Comment:
emm_context does not exist in this case, let's not change this

Reply:
Resolved

Reply:
Resolved